### PR TITLE
Added add and remove features

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Added add todo item feature.
-Added remove item feature.
-Set up the local storage to store to-do lists.

### CODE REVIEW

File: src/storage.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        let indexer = 1;
+        tasks.forEach((elem) => {
+          elem.index = indexer;
+          indexer += 1;
+        });
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+}
```
Comment (Shisui6) [inline src/storage.js:51]: Nice job storing all the add and remove changes in local storage👍. However, according to the project requirements, **all** your app changes should be saved in local storage. This also includes editing a task. Currently this functionality is not implemented. Please add it🙏.

File: src/storage.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      if (tasks.length > 0) {
+        let indexer = 1;
+        tasks.forEach((elem) => {
+          elem.index = indexer;
+          indexer += 1;
+        });
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
```
Comment (Shisui6) [inline src/storage.js:35]: Kindly remove the `>` before the text in your portfolio and tech stack sections in your readme. The `>` makes the line a comment and greys it out, which does not make your readme professional
image

Comment (Shisui6) [inline src/storage.js:35]: Kindly update your getting started section to include updated instructions for how to run your project. As this project makes use of webpack, instructions such as `npm install` and `npm run start` should be included as this is how to start your application
image

Comment (Shisui6) [inline src/storage.js:35]: Kindly remove the `Note:` under your license link. This is simply a note for you the project owner and should not be present for your users to see.
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @addisalemseifu, 

# Requires Changes 🍀
You have done a great job so far!👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🫡

## Highlights
- Add and remove functionality working ✅
- Application saves to local storage ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addisalemseifu** [comment]
- I have also fixed the issue with the README.md file.

**Review by joel-kalema — APPROVED** [review]
Hi @addisalemseifu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**